### PR TITLE
Initial docs for Rewind

### PR DESCRIPTION
**Pull request by p--b**
This PR contains the following commits. You will probably find it easiest to review this
by checking it out locally on your machine and building it. I have taken a somewhat jovial
tone in writing this in the hope that we will amuse our users.

commit db7a66fd3a0bbd5ed984d1d6771cffe063c69a6c (HEAD -> updates, gh/updates)
Author: Pete Bridgman <pb@fourieraudio.com>
Date:   Thu Sep 2 01:16:40 2021 +0100

    README updates
    
    This commit refines the README to make it easier to find the appropriate
    place to read the docs, file issues, or open pull requests.

commit 1cc8ca0648d55fccc6c393bf55596acfebf37e31
Author: Pete Bridgman <pb@fourieraudio.com>
Date:   Thu Sep 2 00:59:28 2021 +0100

    RW-176: Rewind initial documentation
    
    This commit provides initial documentation for Rewind. There's probably
    more to add but this is a starter for 10.

commit c874d5a0595536a7a33d386bd9d801be61e9dcf2
Author: Pete Bridgman <pb@fourieraudio.com>
Date:   Wed Sep 1 20:21:32 2021 +0100

    Remove blogs
    
    Blogs do not belong in the documentation, because it is the
    documentation, not a blog.

commit b211dabfc523ba9afbef6858169ebea542c487bc
Author: Pete Bridgman <pb@fourieraudio.com>
Date:   Wed Sep 1 20:17:34 2021 +0100

    Styling and template updates
    
    This commit makes some minor aesthetic improvements to the style
    and updates a number of the navigation links.

### CODE REVIEW

File: docs/rewind/channel-edit-pane.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+![Channel Edit Pane](/img/rewind/edit-pane.png)
+
+The channel edit pane can be used to configure the channels in your show.
+
+Each channel can optionally be labelled and have a colour assigned. If assigned,
+they will be shown in the [Channel Selection Region](channel-selection-region):
+![Channel Selection Button with label and colour](/img/rewind/ui-channel-button.png).
+
+You can copy and paste channel name data directly into *Rewind* from your favourite
+spreadsheet application. What a truly Excellent feature.
```
Comment (henryharrod) [inline docs/rewind/channel-edit-pane.md:15]:
```suggestion
spreadsheet application.
```

File: src/css/custom.css [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 /* You can override the default Infima variables here. */
 :root {
-  --ifm-color-primary: #25c2a0;
+  --ifm-color-primary: #efff00;
```
Comment (henryharrod) [inline src/css/custom.css:10]:
```suggestion
  --ifm-color-primary: #e2ff00;
```

**Review by henryharrod — CHANGES_REQUESTED** [review]
Great work - just a couple of things :)

File: src/css/custom.css (hunk 2 above)
Comment (henryharrod) [reply src/css/custom.css:10]: I believe our lime green is e2 not ef :)

File: docs/rewind/channel-edit-pane.md (hunk 1 above)
Comment (p--b) [reply docs/rewind/channel-edit-pane.md:15]: I thought this was hilarious :'(

Comment (henryharrod) [reply docs/rewind/channel-edit-pane.md:15]: It's certainly funny if you know how complicated it is to implement ahahahah

Comment (p--b) [reply docs/rewind/channel-edit-pane.md:15]: aha yes dw 'tis cut

**Pull request merged**